main/lsp/requests/document_symbol.cc

```diff
@@ -38,7 +38,13 @@ std::unique_ptr<DocumentSymbol> symbolRef2DocumentSymbol(const core::GlobalState
     auto kind = symbolRef2SymbolKind(gs, symRef);
     // TODO: this range should cover body. Currently it doesn't.
     auto range = loc2Range(gs, sym->loc());
+    if (range == nullptr) {
+        return nullptr;
+    }
     auto selectionRange = loc2Range(gs, sym->loc());
+    if (selectionRange == nullptr) {
```

Comment:
Nit: You can combine these conditionals into `if (range == nullptr || selectionRange == nullptr) { return nullptr; }`.